technical-reports/color/color-type.md

```diff
@@ -1,22 +1,229 @@
-# The color type
+# Color type
 
-Represents a 24bit RGB or 24+8bit RGBA color in the sRGB color space. The `$type` property MUST be set to the string `color`. The value MUST be a string containing a hex triplet/quartet including the preceding `#` character. To support other color spaces, such as HSL, translation tools SHOULD convert color tokens to the equivalent value as needed.
+## Format
 
-For example, initially the color tokens MAY be defined as such:
+Colors can be represented through various formats. For color tokens, the `$type` property MUST be set to the string `color`. The `$value` property can then be used to specify the details of the color, including color space, alpha settings, and more. The `$value` object contains the following properties:
+
+- `$hex` (required): the hex color to use as the default or a guaranteed fallback
```

Comment:
Suggestion: `$hex`, `$colorSpace`, `$name` etc should probably drop the `$` prefix, right? For all other tokens, `$value` objects contain normal non-prefixed keys. Which feels more expected because outside the token, `$value` is a special reserved word that marks any object as a token. But inside the `$value`, a strict schema is enforced where arbitrary properties are no longer allowed (so no benefit to prefixing keys).

Apologies if there’s a discussion I missed or if this suggestion isn’t helpful.